.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,26 @@
+# From: https://levelup.gitconnected.com/enforcing-jira-ticket-formatting-in-github-pr-titles-and-commit-messages-78d9755568df
+name: "PR Title Check"
+on:
+  pull_request:
+    branches: [ dev, LNK-2726-Enforce-Jira-key-in-PR-message ]
+    types: [opened, edited, synchronize]
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
```

Comment:
Consider adding a user-friendly `name`. I assume the "check-title" key is what shows up in the list of "actions" and "checks" for the repository and PRs. If it shows the "name" at the root level, disregard this comment.

Reply:
Disregard, I figured it out. It _does_ show "PR Title Check"